__tests__/books.test.js

```diff
@@ -0,0 +1,34 @@
+const setup = require('../data/setup');
+const app = require('../lib/app');
+const pool = require('../lib/utils/pool');
+const request = require('supertest');
+const Book = require('../lib/models/Book');
+
+describe('books routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  it('should return a list of books with nested authors', async () => {
+    const results = await request(app).get('/books');
+    expect(results.body.length).toEqual(4);
+  });
```

Comment:
Possibly consider revising this test, as if data is added, the test will fail. Consider testing the shape of the data with .toEqual(expect.arrayContaining([{...}])